### PR TITLE
dependencies

### PR DESCRIPTION
**Pull request by Rexogamer**

### CODE REVIEW

**Comment by Rexogamer** [comment]
i do feel the discord.js one might be wrong but blame dependabot for that if so

**Comment by Androz2091** [comment]
The latest release of Discord js is 11.5 I don't think 9.3 is a good version to use yet...

**Comment by Rexogamer** [comment]
I fixed it.

**Comment by Rexogamer** [comment]
@Androz2091

**Comment by Androz2091** [comment]
All right! Since there are many differences between the dependencies of your pull request and the current ones, you should do a whole series of tests to check that everything is working well. I think you didn't do it otherwise it would have been a problem with discord.js v9... This is important.

**Comment by Rexogamer** [comment]
I’m sorry but I’m unable to test it myself (kinda don’t know how/don’t really have the hardware for it); if someone else with hosting abilities would be able to use the branch files to test it then sure but I sadly cannot

**Comment by Androz2091** [comment]
Okay. Because it's really important it can break the whole bot...

**Comment by Rexogamer** [comment]
i resovled the confilct; now can someone test this?

**Comment by F9Alejandro** [comment]
@Rexogamer I will test it for you, please next time attempt to get the bot up and running on your system for testing instead of having others do it for you.

**Comment by Rexogamer** [comment]
> @Rexogamer I will test it for you, please next time attempt to get the bot up and running on your system for testing instead of having others do it for you.

I'm sorry that half the time I use an iPad and didn't spend forever installing dependencies!

**Comment by Rexogamer** [comment]
@F9Alejandro In nicer terms, I don't have the hardware to run anything required for this bot, and didn't wish to spend at least 2 days installing everything (as I only have limited time on my laptop as it is a school one)

**Comment by F9Alejandro** [comment]
@Rexogamer Sorry, I didn't know. Would you like to have this put in its own branch? I ask because some people would like to keep using the old Discord.js v9.3 instead of Discord.js v11.5, or they have scripts/plugins that would break in the newer version.

**Comment by Rexogamer** [comment]
@F9Alejandro sure

File: package.json [hunk 1]
```diff
@@ -40,6 +43,8 @@
     "wikijs": "^5.5.0",
     "youtube-dl": "^2.0.0",
     "youtube-node": "^1.3.3"
```
Comment (F9Alejandro) [inline package.json]: Missing Comma

Comment (Rexogamer) [reply package.json]: added

**Comment by F9Alejandro** [comment]
Created Branch **rexo/discord-11.5** for this merge, difference between master and new branch is discord.js version and updated dependencies.

File: package.json [hunk 2]
```diff
@@ -42,9 +42,9 @@
     "urban": "^0.3.1",
     "wikijs": "^5.5.0",
     "youtube-dl": "^2.0.0",
-    "youtube-node": "^1.3.3"
+    "youtube-node": "^1.3.3",
     "ytdl-core": "^0.29.3",
-    "zucc": "^0.1.2"
+    "zucc": "^0.1.2",
```
Comment (F9Alejandro) [inline package.json]: don't need a comma for this one as it ends an object, will fix on chalda:rexo/discord-11.5 branch

**Pull request merged**